react/fields/FieldLabel/FieldLabel.less

```diff
@@ -0,0 +1,11 @@
+@import (reference) "~seek-style-guide/theme";
+
+.label {
+  @media only screen and (max-width: (@responsive-breakpoint - 1)) {
+    font-weight: @sk-bold;
+  }
+}
+
+.secondaryLabel {
+  font-weight: @sk-light;
```

Comment:
secondaryLabel is really a internal/private implementation used for the max character wording in the TextArea however I will run it by Steph.